x-pack/filebeat/module/checkpoint/firewall/ingest/pipeline.yml

```diff
@@ -40,10 +40,10 @@ processors:
     - message
     - host
     ignore_missing: true
-- rename:
-    field: "@timestamp"
-    target_field: "event.created"
-    ignore_missing: true
+- set:
+    copy_from: "@timestamp"
+    field: "event.created"
+    ignore_empty_value: true
```

Comment:
ignore_empty_value only has an effect when using `value`. With copy from you'd need an `if` to achieve a similar behavior.

Reply:
After sleeping on it, I realized that we should not ignore empty values. If `@timestamp` is missing that is an error.